packages/admin-portal/src/components/ui/diff-viewer/diff-viewer.tsx

```diff
@@ -17,15 +18,15 @@ const diffTypes = {
   wordsWithSpace: jsdiff.diffWordsWithSpace,
 }
 
-const DiffViewer = ({ currentString, changedString, type = 'words' }: DiffViewerProps) => {
+export const DiffViewer = ({ currentString, changedString, type = 'words' }: DiffViewerProps) => {
   const result = diffTypes[type](currentString, changedString).map((part, index) => {
     return (
-      <span key={index} className={part.added ? styles.green : part.removed ? styles.red : ''}>
+      <span key={index} className={cx(part.added ? greenBackground : part.removed && redBackground)}>
```

Comment:
Could we change to `cx(part.added && greenBackground, part.removed && redBackground)` to avoid if else